src/utils/isDayVisible.js

```diff
@@ -21,7 +21,7 @@ export default function isDayVisible(day, month, numberOfMonths, enableOutsideDa
 
   if (enableOutsideDays) {
     if (!startCacheOutsideDays.has(startKey)) {
-      startCacheOutsideDays.set(startKey, month.clone().startOf('month').startOf('week'));
+      startCacheOutsideDays.set(startKey, month.clone().startOf('month').startOf('week').hour(12));
```

Comment:
same question here

Reply:
Same as for ```getNumberOfCalendarMonthWeeks```, just to honor unified approach to get first day of week/month.